angular/src/app/account/send-sidechain/address/send-address.component.ts

```diff
@@ -37,13 +37,26 @@ export class AccountSendSidechainAddressComponent implements OnInit, OnDestroy {
     private ngZone: NgZone,
     public dialog: MatDialog,
     private fb: UntypedFormBuilder
-  ) {
-    this.form = fb.group({
+  ) {}
+
+  ngOnDestroy() {}
+
+  async ngOnInit() {
+    // Constructor of send.component.ts should have called resetFee() by now, which sets
+    // the fee to network definition. Here we will attempt to get it from blockchain API.
+    const networkStatus = this.networkStatusService.get(this.sendService.network.id);
+
+    // Grab the fee rate either from network definition or from blockchain API status:
+    if (networkStatus.length > 0) {
```

Comment:
In the future fee relay should probably be broken down to a list of fee/block-count items

Reply:
What if networkStatus.length = 0 fee will be zero?

Reply:
If we don't have network status, the fee is taken from our network types, like `STRAX.ts` which has defined minFeeRate and maxFeeRate.